project_clusters.go

```diff
@@ -115,12 +127,13 @@ func (s *ProjectClustersService) GetCluster(pid interface{}, cluster int, option
 // GitLab API docs:
 // https://docs.gitlab.com/ee/api/project_clusters.html#add-existing-cluster-to-project
 type AddClusterOptions struct {
-	Name               *string                       `url:"name,omitempty" json:"name,omitempty"`
-	Domain             *string                       `url:"domain,omitempty" json:"domain,omitempty"`
-	Enabled            *bool                         `url:"enabled,omitempty" json:"enabled,omitempty"`
-	Managed            *bool                         `url:"managed,omitempty" json:"managed,omitempty"`
-	EnvironmentScope   *string                       `url:"environment_scope,omitempty" json:"environment_scope,omitempty"`
-	PlatformKubernetes *AddPlatformKubernetesOptions `url:"platform_kubernetes_attributes,omitempty" json:"platform_kubernetes_attributes,omitempty"`
+	Name                *string                       `url:"name,omitempty" json:"name,omitempty"`
+	Domain              *string                       `url:"domain,omitempty" json:"domain,omitempty"`
+	Enabled             *bool                         `url:"enabled,omitempty" json:"enabled,omitempty"`
+	Managed             *bool                         `url:"managed,omitempty" json:"managed,omitempty"`
+	EnvironmentScope    *string                       `url:"environment_scope,omitempty" json:"environment_scope,omitempty"`
+	PlatformKubernetes  *AddPlatformKubernetesOptions `url:"platform_kubernetes_attributes,omitempty" json:"platform_kubernetes_attributes,omitempty"`
+	ManagementProjectID *string                       `url:"management_project_id,omitempty" json:"management_project_id,omitempty"`
```

Comment:
According to the docs this isn't a valid option for the API call.

Reply:
The object was added with https://gitlab.com/gitlab-org/gitlab/-/merge_requests/18429